### ISSUE TITLE
Support of the IBM Power9 (ppc64le) architecture

### ISSUE THREAD

**Issue by raimis** [conda-forge/openmm-plumed-feedstock#10]
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

*PLUMED* has been ported to `ppc64le` (https://github.com/conda-forge/plumed-feedstock/issues/22, https://github.com/conda-forge/plumed-feedstock/pull/23).

So, we can port *OpenMM-PLUMED* too! @jaimergp do you foreseen any problems?

**Comment by jaimergp** [comment]
Yes, it's possible, but right now building PPC packages with CUDA support involves changing to manual feedstock maintenance (see [this file at OpenMM's feedstock](https://github.com/conda-forge/openmm-feedstock/blob/master/recipe/conda_build_config.yaml)), and I'd rather not maintain too many feedstocks this way.

You can build it locally if you adapt [this file](https://github.com/conda-forge/openmm-feedstock/blob/master/.ci_support/linux_ppc64le_cuda_compiler_version10.2numpy1.17python3.6.____cpythonpython_implcpython.yaml) from OpenMM's feedstock and drop it in your local copy of this feedstock. Then `python build_locally.py` should give you a tarball you can distribute.

If that's not a viable solution, we can discuss it further but I can't promise much.

**Comment by raimis** [comment]
Thanks @jaimergp, I will try the manual solution to test.

When we will be able to build the PPC packages with CUDA properly? Is any issue to check the progress?

**Comment by jaimergp** [comment]
I'd say it has to do with the CentOS 8 issue here: https://github.com/conda-forge/conda-forge.github.io/issues/1432

**Comment by raimis** [comment]
@jaimergp any updates on this?

**Comment by jaimergp** [comment]
Nothing new. There's a solution we can try and I am going to put some time there, we'll see how it works in practice. https://github.com/conda-forge/cudatoolkit-feedstock/issues/64
